.github/workflows/multichain-e2e-template.yml

```diff
@@ -0,0 +1,106 @@
+name: Multichain E2E Workflow Template
+
+on:
+  workflow_call:
+    inputs:
+      config:
+        required: true
+        type: string
+      relayer-type:
+        required: true
+        type: string
+      registry-port:
+        required: true
+        type: number
+
+jobs:
+  multichain-e2e:
+    runs-on: ubuntu-latest-16core
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: 'true'
+          path: ./agoric-sdk
+      - uses: ./agoric-sdk/.github/actions/restore-node
+        with:
+          node-version: 18.x
+          path: ./agoric-sdk
+
+      - name: yarn link
+        run: |
+          yarn link-cli ~/bin/agoric
+          echo "/home/runner/bin" >> $GITHUB_PATH
+        working-directory: ./agoric-sdk
+
+      - name: Enable Corepack
+        run: corepack enable
+        working-directory: ./agoric-sdk/multichain-testing
+
+      - name: Install @agoric/multichain-testing deps
+        run: yarn install
+        working-directory: ./agoric-sdk/multichain-testing
+
+      - name: Lint @agoric/multichain-testing
+        run: yarn lint
+        working-directory: ./agoric-sdk/multichain-testing
+
+      - name: Setup Starship Infrastructure
+        id: starship-infra
+        uses: cosmology-tech/starship-action@0.3.0
+        with:
+          # uses ghcr.io/agoric/agoric-sdk:dev image (latest master)
+          values: ./agoric-sdk/multichain-testing/${{ inputs.config }}
+          port-forward: false
```

Comment:
I see in the commit this is just until https://github.com/cosmology-tech/starship/pull/561 is available.

That's helpful but even more important is for the source to say so, to anyone browsing who could then simplify this code.

Incidentally, isn't your PR available now in a starship release?